algos.yaml

```diff
@@ -90,6 +110,17 @@ deep-10M:
               "nprobe=2,quantizer_efSearch=8",
               "nprobe=4,quantizer_efSearch=4",
               "nprobe=2,quantizer_efSearch=16"]
+    diskann-t2:
```

Comment:
There were two sections called "deep-10M" so I just moved the "diskann-t2" section up here to deduplicate the sections.